src/encoded/static/components/viz/Workflow/Graph.js

```diff
@@ -110,25 +112,30 @@ export default class Graph extends React.Component {
             });
             return nodes;
         },
+        /**
+         * This function, along with 'nodesInColumnPostSortFxn' & 'nodesPreSortFxn', is _PROTOTYPICAL_ - a R&D work in progress.
+         * It's quick, dirty, hacky. And it will remain this way until we figure out what we want from it, or if we want it at all, and then we'll go back and make it DRY and pretty.
+         */
         'nodesInColumnSortFxn' : function(node1, node2){
```

Comment:
This part _should_ be cleaned up at some point, but it won't for a while because is just experimental row positioning at this point. If find approach we like, will then cement/clean/refactor.